src/plugins/kibana/public/visualize/editor/agg_group.html

```diff
@@ -5,7 +5,7 @@
 
   <div class="vis-editor-agg-group" ng-class="groupName">
     <!-- wrapper needed for nesting-indicator -->
-    <div ng-repeat="agg in group" class="vis-editor-agg-wrapper">
+    <div class="vis-editor-agg-wrapper" ng-repeat="agg in group">
```

Comment:
Ah, I guess I'm just used to having them the other way around (FWIW the `<div` right above this line is consistent with the change I had made. But good to know